### PR TITLE
Feature/SK-823 | Hugging Face Example

### PR DESCRIPTION
**Pull request by FrankJonasmoelle**
FEDn example that uses the Hugging Face Transformer library

### CODE REVIEW

File: examples/huggingface/README.rst [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+- Go to the 'Sessions' menu, click on 'New session', and upload the compute package (package.tgz) and seed model (seed.npz).
+- In the 'Clients' menu, click on 'Connect client' and download the client configuration file (client.yaml)
+- Save the client configuration file to the huggingface example directory (fedn/examples/huggingface)
+
+To connect a client, run the following command in your terminal:
+
+.. code-block::
+
+   fedn run client -in client.yaml --secure=True --force-ssl
```
Comment (Wrede) [inline examples/huggingface/README.rst:78]: use "fedn client start" instead

File: examples/huggingface/README.rst [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+Alternatively, if you prefer to use Docker, run the following:
+
+.. code-block::
+
+   docker run \
+   -v $PWD/client.yaml:/app/client.yaml \
+   -e CLIENT_NUMBER=0 \
+   -e FEDN_PACKAGE_EXTRACT_DIR=package \
+   ghcr.io/scaleoutsystems/fedn/fedn:0.9.0 run client -in client.yaml --secure=True --force-ssl
```
Comment (Wrede) [inline examples/huggingface/README.rst:89]: "client start"

File: examples/huggingface/docker-compose.override.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+services:
+
+  client1:
+    extends:
+      file: ${HOST_REPO_DIR:-.}/docker-compose.yaml
+      service: client
+    environment:
+      <<: *defaults
+      FEDN_DATA_PATH: /app/package/data/clients/1/enron_spam.pt
```
Comment (Wrede) [inline examples/huggingface/docker-compose.override.yaml:19]: think it should be /app/package/client/data... with recent changes comming from master

File: examples/huggingface/docker-compose.override.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    volumes:
+      - ${HOST_REPO_DIR:-.}/fedn:/app/fedn
+
+  client2:
+    extends:
+      file: ${HOST_REPO_DIR:-.}/docker-compose.yaml
+      service: client
+    environment:
+      <<: *defaults
+      FEDN_DATA_PATH: /app/package/data/clients/2/enron_spam.pt
```
Comment (Wrede) [inline examples/huggingface/docker-compose.override.yaml:31]: /app/package/client/data

**Review by Wrede — CHANGES_REQUESTED** [review]

**Review by ahellander — APPROVED** [review]

**Review by Wrede — APPROVED** [review]

**Pull request merged**